index.html

```diff
@@ -0,0 +1,58 @@
+<!-- DOCTYPE html -->
+<html lang="en-GB">
+	
+	<head>
+		<link rel="stylesheet" type="text/css" href="normalize.css">
+		<link rel="stylesheet" type="text/css" href="styles.css">
+	</head>
+	
+	<body>
+		<nav>
+			<div>
+				<ul>
+					<li>about me</li>
+					<li>portfolio</li>
+					<li>contact me</li>
+				</ul>	
+			</div>	
+		</nav>
+	
+		<header>
+			<section class="container">
+				<h1>Charlotte Nightingale</h1>
+				<h2>full stack developer</h2>
+			</section>
+		</header>
+		
+			<div class="portfolio">
+				
+						
+				<div class="project" id="projectOne">
+				<h3>Spinnning a web</h3>
+				</div>
+						
+				<div class="project" id="projectTwo">
+				<h3>The bees' knees</h3>
+				</div>
+							
+				<div class="project" id="projectThree">
+				<h3>On fire, no bugs</h3>
+				</div>
+	
+			
+			</div>
+			
+			<section class="container">
+				<h1>About me:</h1>
+				<p>This is where I'm going to tell you all about myself and the skills that I am developing on the Full Stack Track at Mayden Academy.  There's the minor detail of the online test, and then I'll be a certified scrum master, which I'll tell you about too.  I'm based in Bath and am looking for my first role as a junior developer: I'm available from the middle of December 2019.</p>
+			</section>
+			
+			<footer>
+				
+				<p>Contact me: <a href="mailto:charlottenightingale@ymail.com">
+				  charlottenightingale@ymail.com</a>.</p>
```

Comment:
don't need the full stop between the a and p tags. If you leave it in, people might copy/paste your email address into their mail client and scoop up the dot, and the email won't reach you